azure-pipelines.yml

```diff
@@ -408,10 +408,18 @@ jobs:
     parameters:
       downloadPartialBuildBinFrom:
         - cw310_sw_build
+  - bash: |
+      ci/scripts/get-bitstream-strategy.sh '^sw\/' 'testplan\.hjson$' '^site\/' '^doc\/' '^COMMITTERS' '^CLA' '\.md$'
```

Comment:
Per the offline discussion, we also mentioned adding `hw\/dv` and `hw\/.+\/dv`. For DV changes, we probably don't even need to run the FPGA tests at all. Cleaning up the FPGA testing conditions is covered in #14400, but maybe we can add these conditions here for now.

Reply:
Done